README.md

```diff
@@ -583,6 +583,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager        |
 |------------------------|----------------------|---------------------|
+| v0.81.0                | v1.19 to v1.27       | v1                  |
```

Comment:
please remove the last one

Reply:
Updated.  Also updated the release instructions with this detail.